propolis/src/hw/chipset/i440fx.rs

```diff
@@ -678,6 +678,8 @@ impl Piix3PM {
             ..Default::default()
         })
         .add_custom_cfg(PMCFG_OFFSET as u8, PMCFG_LEN as u8)
+        // ACPI device requires lintr for SCI
+        .add_lintr()
```

Comment:
Prior to adding this, `lspci` (on alpine) knew of the interrupt associated with the device thanks to the ACPI table, but was still confused about it due to the missing entry here:
```
Interrupt: pin ? routed to IRQ 9
```

With the entire change, `lspci` is much happier about reporting all of the values (for an attached virtio disk and nic):
```
localhost:~# lspci -vv | grep Interrupt
        Interrupt: pin A routed to IRQ 9
        Interrupt: pin B routed to IRQ 10
        Interrupt: pin B routed to IRQ 11
```